### ISSUE TITLE
Blur Effects

### ISSUE THREAD

**Issue by jsmorabito** [penpot/penpot#1185]
**Is your feature request related to a problem? Please describe.**
I want to make hi fidelity mock ups for Mac applications, but cannot create the transparent, blurry background window effect

**Describe the solution you'd like**
Layer blur and background blur like Figma

**Additional context**
https://help.figma.com/hc/en-us/articles/360041488473-Apply-shadow-or-blur-effects

**Comment by claragvinola** [comment]
Hi Igorgro! thank you so much for your suggestion! We will study and evaluate it :)

**Comment by claragvinola** [comment]
https://tree.taiga.io/project/penpot/issue/2024

**Comment by exellian** [comment]
Are there any plans that this feature will get implemented?

**Comment by astudentinearth** [comment]
Came here after trying to prototype a blurred panel. Are there any plans for implementing backdrop blur?

**Comment by superalex** [comment]
Sure, we have the plans, we just need more time :)

You can track all penpot project developtment [here](https://tree.taiga.io/project/penpot/timeline)

**Comment by corneliusb79** [comment]
+1

This issue is super important to my design work as we use the glass effect a lot which is currently keeping us between Penpot and Figma.

**Comment by astudentinearth** [comment]
That issue has been around for a while, but I believe the team has other priorities than adding effects. They recently gave us flex layout which was a game changer for me

**Comment by raluvy95** [comment]
Any updates? Figma just put developer mode behind paywall and we are looking for open source alternative to Figma.

**Comment by KailunLarks** [comment]
Also focus on this... Any updates? The issue has stuck on taiga for almost 2 years! [Taiga Link](https://tree.taiga.io/project/penpot/issue/2024)

**Comment by BraveDevelopment** [comment]
> Also focus on this... Any updates? The issue has stuck on taiga for almost 2 years! [Taiga Link](https://tree.taiga.io/project/penpot/issue/2024)

YES PLEASE!!! Thats the only thing thats holding me back from using Penpot. All of my designs need backdrop blur sooo please add it.

**Comment by ChrisNSki** [comment]
Having to use Figma in place of this until it is available, would love to see this added in!

**Comment by nahiiko** [comment]
3 years... ouch

Sadly don't have time to implement this myself, will have to go back to Figma, like many.

Hopefully implemented soon ! much love to the team.

**Comment by TheLostProgrammer** [comment]
Would really love to have this feature on Penpot as it is one of the few final features holding me down to Figma. Figma is getting worse and worse since Adobe acquired it, so really hope Penpot can add features like background blur.

**Comment by WerySkok** [comment]
Hello, it'd be a really nice effect to see, and it can be extended withthe new Liquid Glass effect recently presented by Apple

**Comment by romiem** [comment]
In case others haven't seen, it seems background blur isn't coming until the new render engine is finalised:
https://community.penpot.app/t/why-doesn-t-penpot-have-background-blur-yet/9782

Please please please prioritise shipping this. My designs make extensive use of backdrop blur and I think you will have a tonne of people jumping ship from Figma to Penpot once this feature is in.

Pretty please 😁

**Comment by Beryesa** [comment]
Yay https://github.com/penpot/penpot/pull/8656 thanks for the update!
